PCU/src/main/java/com/coffeelovers/pcu/controller/ProductUnitCostController.java

```diff
@@ -103,6 +103,16 @@
 		}		
 	}
 
+	@GetMapping("/variableCosts")
```

Comment:
let's check the name convention
```suggestion
	@GetMapping("/variableCosts")
```